modules/nw-configuring-lb-allowed-source-ranges.adoc

```diff
@@ -0,0 +1,30 @@
+// Modules included in the following assemblies:
+//
+// * networking/configuring_ingress_cluster_traffic/configuring-ingress-cluster-traffic-load-balancer-allowed-source-ranges.adoc
+
+:_content-type: PROCEDURE
+[id="nw-configuring-lb-allowed-source-ranges_{context}"]
+= Configuring Load Balancer allowed source ranges
+
+By configuring Load Balancer allowed source ranges, you can limit the access to the Load Balancer for the `ingresscontroller` to a specified list of IP address ranges. The Ingress Operator reconciles the Load Balancer Service and sets the `.Spec.LoadBalancerSourceRanges` based on `AllowedSourceRanges`.
```

Comment:
* Should `ingresscontroller` just be "Ingress Controller"?
* Also add a "field" or something after the `.Spec.LoadBalancerSourceRanges`
* I'd also consider changing everywhere in this PR where you talk about `.Spec.LoadBalancerSourceRanges` and `AllowedSourceRanges` to camelcase, like  `spec.loadBalancerSourceRanges` and `allowedSourceRanges` to match how you actually see it in your YAML in the procedures. I think the capital first letter is only shown that way if you do `oc describe`. Also suggest removing the "." in front of "spec", because I don't think it is actually a level down

Reply:
Addressed